README.md

```diff
@@ -307,6 +308,14 @@ They can be added to a scenario file like this:
         tasks = ["assemble"]
 
         apply-build-script-change-to = "build.gradle.kts"
+        apply-project-dependency-change-to {
+            files = ["build.gradle"]
+            # Default number of applied-projects-count is 3.
+            # Gradle-profiler will generate minimal number of projects to have a unique combination of project dependencies every run.
+            # Note: Number of projects is calculated as binomial coffiecient: "X choose applied-projects-count = iterations * files",
+            # due to that, higher the applied-projects-count is, less projects will be generated.
```

Comment:
```suggestion
            # Because of this the higher the applied-projects-count is the fewer projects will be generated.
```

Reply:
I removed that "because of this..." line, it seems confusing. If someone will want use some different non-default value, he/she can calculate values with provided formula.